### DISCUSSION TITLE
What is the motivation behind deprecating many of functions in 1.0 release?

### DISCUSSION THREAD

**Discussion by vovavili** [pyjanitor-devs/pyjanitor#1240] [General]
According to the API reference, many of pyjanitor's functions are to be deprecated in the 1.0 release in favor of either pandas' native methods or pyjanitor's transform_columns. In my personal subjective taste, I prefer to use janitor's more descriptive names in my ETL scripts than terse equivalents. 'add_columns' just reads nicer than 'assign'.

Is there a good reason to deprecate so many functions in the 1.0 release?

**Comment by samukweku** [comment]
Hi @vovavili apologies for the late response.  Pyjanitor tries not to replicate functions that already exist in Pandas. `add_columns` may read nicer, yet it does the same thing as  `assign`. You can always create a function and use pipe, or create a method with `pandas_flavor`

Thanks for the feedback - we do appreciate it
